test/manual/keychain_provider/test_config

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
this file name makes it sound like it _is_ the config, whereas in reality it's setting up the environment with the config.  

i'd change to something like `create_test_env_vars`

Reply:
went with `load_test_env_vars`